app/terminal/terminal.py

```diff
@@ -100,16 +101,23 @@ def do_st(self, op_id):
         service = self._locals['services']['operation_svc']
         asyncio.run_coroutine_threadsafe(service.run(op_id), loop=self._loop)
         self._sout.write(str(colored('Started!', 'yellow')) + '\n')
+	
+	def add_facts_from_csv(self, csv_path, table_name, op_id):
```

Comment:
let's rename this with a _ prefix, to make it more visible that it is private. then move it to the bottom, where other private functions live

Reply:
This should be done in the latest commit